backend/pipeline/serializers/crud.py

```diff
@@ -73,6 +74,20 @@ def _add_connector_data(
                         connector_id=instance.connector_id,
                     )
                 )
+            if repr[PC.DESTINATION_NAME] == PC.NOT_CONFIGURED:
+                try:
+                    check_manual_review = WorkflowEndpoint.objects.get(
+                        workflow=instance.workflow,
+                        endpoint_type=WorkflowEndpoint.EndpointType.DESTINATION,
+                        connection_type=WorkflowEndpoint.ConnectionType.MANUALREVIEW,
+                    )
+                    if check_manual_review:
+                        repr[PC.DESTINATION_NAME] = (
+                            WorkflowEndpoint.ConnectionType.MANUALREVIEW
+                        )
+                except Exception as ex:
+                    logger.debug(f"Not a Manual review destination: {ex}")
```

Comment:
@vishnuszipstack assume that we expect a manual review destination to be present however there's a genuine exception scenario - we'll actually not even raise it properly and continue.
Instead of checking whether its possibly this type - I think we should make a strict check on the connector type or any other entity to denote that its actually MRQ

Reply:
Discussed this separately - given how manual review is not associated with a connector explicitly such checks might not be possible